src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,396 @@
+
+<!---
+The following link is temporary and points to the bep031 branch of bids-examples,
+update to 'https://github.com/bids-standard/bids-examples' when examples are merged.
+-->
+Microscopy datasets formatted using this specification are available on the
+[BIDS examples repository](https://github.com/neuropoly/bids-examples/tree/bep031)
+(`microscopy_SEM001` and `microscopy_SPIM001`) and can be used for practical
+guidance when curating a new dataset.
+
+Further Microscopy datasets are available:
+
+-   In PNG format:  [`data_axondeepseg_sem`](https://doi.org/10.5281/zenodo.5498378)
+-   In OME-TIFF format: [`Broca's Area Light-Sheet Microscopy`](https://doi.org/10.5281/zenodo.5517223)
+
+## Microscopy recording data
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+Template:
+```Text
+sub-<label>/
+    [ses-<label>/]
+        microscopy/
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.<extension>
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.json
+```
+Microscopy data MUST be stored in the `microscopy` directory.
+
+### File formats
+The Microscopy community uses a variety of formats for storing raw data, and there is no single
+standard that all researchers agree on. However, a standardized file structure has been developed
+by the [Open Microscopy Environment](https://www.openmicroscopy.org/) for whole-slide imaging with
+the [OME-TIFF file specifications](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/file-structure.html).
+The OME-TIFF file allows for multi-page TIFF files to store multiple image planes and supports
+multi-resolution pyramidal tiled images. An OME-XML data block is also embedded inside the
+file’s header.
+
+In Microscopy-BIDS, we accept different file formats to accommodate datasets stored in 2D image
+formats and whole-slide imaging formats, to accommodate lossless and lossy compression,
+and to avoid unnecessary conversions of the original data from a non-tiled to a tiled format,
+or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions of Microscopy-BIDS may extend this list of supported file formats.
```

Comment:
Again, `Microscopy-BIDS` is awkward.  Maybe just omit here:

> Future versions may extend this list...

Reply:
Thank you @nicholst for your review and feedback!
I have integrated both your suggestions.